firebase-abt/firebase-abt.gradle

```diff
@@ -17,7 +17,7 @@ plugins {
 }
 
 firebaseLibrary {
-    testLab.enabled = true
+    testLab.enabled = false
```

Comment:
It's false by default, so should be able to just remove this line, up to you

Reply:
I think defaulting to true might be better behavior. This way, it's explicit that the tests are not running (in case they add some in the future).